integration-tests/maven/src/test/java/io/quarkus/maven/it/JarRunnerIT.java

```diff
@@ -341,4 +307,53 @@ private void dumpFileContentOnFailure(final Callable<Void> operation, final File
             throw t;
         }
     }
+
+    private void assertThatFastJarFormatWorks(String outputDir) throws Exception {
+        File testDir = initProject("projects/classic", "projects/project-classic-console-output-fast-jar");
+        RunningInvoker running = new RunningInvoker(testDir, false);
+
+        MavenProcessInvocationResult result = running
+                .execute(Arrays.asList("package",
+                        "-DskipTests",
+                        "-Dquarkus.package.type=fast-jar",
+                        outputDir == null ? "" : "-Dquarkus.package.output-directory=" + outputDir), Collections.emptyMap());
+
+        await().atMost(1, TimeUnit.MINUTES).until(() -> result.getProcess() != null && !result.getProcess().isAlive());
+        assertThat(running.log()).containsIgnoringCase("BUILD SUCCESS");
+        running.stop();
+
+        Path jar = testDir.toPath().toAbsolutePath()
+                .resolve(Paths.get("target/acme-1.0-SNAPSHOT-runner.jar"));
+        Assertions.assertFalse(Files.exists(jar));
+
+        jar = testDir.toPath().toAbsolutePath()
+                .resolve(Paths.get("target",
+                        outputDir == null ? JarResultBuildStep.DEFAULT_FAST_JAR_DIRECTORY_NAME : outputDir,
```

Comment:
Instead of passing `null`, why not pass `JarResultBuildStep.DEFAULT_FAST_JAR_DIRECTORY_NAME` directly?

Reply:
Because you also want to test what would happen if nothing is passed in.